frontend/src/scenes/onboarding/OnboardingProductConfiguration.tsx

```diff
@@ -14,53 +42,88 @@ export const OnboardingProductConfiguration = ({
     options: ProductConfigOption[]
 }): JSX.Element | null => {
     const { configOptions } = useValues(onboardingProductConfigurationLogic)
+    const { defaultEnabledPlugins } = useValues(pluginsLogic)
     const { setConfigOptions, saveConfiguration } = useActions(onboardingProductConfigurationLogic)
+    const { toggleEnabled } = useActions(pluginsLogic)
+
     useEffect(() => {
         setConfigOptions(options)
     }, [])
 
-    return configOptions ? (
+    const combinedList: ConfigOption[] = [
+        ...configOptions.map((option) => ({
+            title: option.title,
+            description: option.description,
+            type: option.type as ConfigType,
+            selectOptions: option.selectOptions,
+            value: option.value,
+            onChange: (newValue: boolean | string | number) => {
+                setConfigOptions(
+                    configOptions.map((o) => (o.teamProperty === option.teamProperty ? { ...o, value: newValue } : o))
+                )
+            },
+        })),
+        ...defaultEnabledPlugins.map((plugin) => {
+            const pluginContent = pluginContentMapping[plugin.id]
```

Comment:
If the plugin doesn't exist in the mapping this will probably return a bunch of null values. Not something a user is likely to see but we can still check if it exists and if not just return `undefined`, then filter out the `undefined`s.

Reply:
The map is an object so it will return null and lines 69/70 will fallback to the actual plugin name/description.